examples/popover/popover.controlled.tsx

```diff
@@ -0,0 +1,104 @@
+'use client';
+
+import { useState } from 'react';
+import { BiCaretDown } from 'react-icons/bi';
+import { type CodeData } from '~/components/code-demo';
```

Comment:
using absolute imports (eg: `~/...`) will break the library, unfortunately.

Reply:
@SutuSebastian this is in the doc examples for the documentation app. I checked the other examples and they all use the absolute imports.

However, my import paths in the Popover component itself were using absolute paths. I made the fix only in the Popover components and left the absolute imports in the docs examples (I tried using relative paths but the Vercel deployment was failing).

Reply:
Ah yeah, right, sorry! 😄 But anything in `src` folder should have relative imports because `tsc` does not bundle anything